src/test/resources/features/mainPageTests.feature

```diff
@@ -0,0 +1,122 @@
+@mainPageTests
```

Comment:
Finally separate feature files - but all the features need annotation `@apicuritoTests` otherwise they would run in syndesis tests.

Reply:
updated